integration-tests/test_run_one_off_build.py

```diff
@@ -0,0 +1,49 @@
+from os import environ
+from unittest.mock import patch
+
+import pytest
+
+from metaci.build.management.commands.run_build import make_build
+from metaci.build.management.commands.run_build_from_id import Command
+from metaci.conftest import (
+    BranchFactory,
+    OrgFactory,
+    PlanFactory,
+    PlanRepositoryFactory,
+    RepositoryFactory,
+    UserFactory,
+)
+
+
+@pytest.mark.django_db
+@patch("metaci.build.tasks.reset_database_connection", lambda: ...)
+class TestOneOffBuild:
+    @patch.dict(environ, {"LOG_TO_STDERR": "True"})
+    def test_one_off_build(self, capsys):
+        repo = RepositoryFactory(
+            name="CumulusCI-Test",
+            owner="SFDO-Tooling",
+            url="https://github.com/SFDO-Tooling/CumulusCI-Test",
+        )
+        OrgFactory(name="myorg", repo=repo, scratch=False)
+        plan = PlanFactory(
+            name="myplan",
+            org="myorg",
+            queue="long-running",
+            flows="ci_test_concurrency",
+        )
+        BranchFactory(name="main", repo=repo)
+        PlanRepositoryFactory(repo=repo, plan=plan)
+        UserFactory(username="fake_username")
+        build = make_build(
+            "CumulusCI-Test",
+            "main",
+            "main",
+            "myplan",
+            "fake_username",
+        )
+        c = Command()
+        c.handle(build_id=build.id, lock_id="abcde")
+        outerr = capsys.readouterr()
+        print(outerr.err)
+        assert "ci_test_concurrency" in outerr.err + outerr.out
```

Comment:
I'm still not clear on the goal of this test. If you want to make sure the run_build_from_id command ran successfully, looking for the flow name in the output doesn't seem like a good indicator of that.

I can see the rationale for having an integration test that actually fetches the commit from github, but I'm not thrilled about adding a 60-second sleep to every MetaCI test run.

Reply:
Ultimately, before I deploy a feature I want to see it run on my computer. This is a one-command way of seeing the thing work with my own eyes. I added it to the Github action because I anticipated a potential concern that a test that isn't run often will tend to rot. The integration test does not run locally when you just run 'pytest'. I don't feel strongly about whether it should or shouldn't run on every push to github, but I see relatively minor harm in it.

I could make the condition it tests for (slightly) more robust but the build doesn't execute all of the way to the end due to key management issues which I don't have the time to debug and are fairly orthogonal to what I want to test.

Reply:
I made the assertion slightly more robust.

Also, the test only takes 11 seconds to run, including the 8-9 seconds of startup time for MetaCI tests in general, so I don't think there is any significant cost to it.

Reply:
@prescod What sort of key management issues? But it sounds like it's a test for confirming that the build started, not that it ran to completion.